test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+
+if [ -f /usr/bin/python3 ]
+then
+    PYTHON="--python /usr/bin/python3"
+else
+    PYTHON=""
+fi
+
+virtualenv --system-site-packages $PYTHON "${MYTMPDIR}/jinja2"
+
+source "${MYTMPDIR}/jinja2/bin/activate"
```

Comment:
Ah, sorry I wasn't clear. I didn't mean to change how you're selecting the python version. I just meant that it would be good to run `which python` after activating the venv, so it was clear which python was now used.

I actually need to improve `ansible-test` a bit in this area, as using `which python` has some odd behavior with the `--python` option, but that's something I'll deal with later.